ansible/playbooks/vars/docker-deploy.yml

```diff
@@ -5,4 +5,7 @@ launch_hand: false
 product: ''
 ethercat_interface: "eth0"
 use_aws: false
-customer_key: false
\ No newline at end of file
+customer_key: false
+unit_tests: false
+user_folder: "/home/{{ lookup('env','USER') }}"
```

Comment:
this should not be here. It should be in unit-test playbook

Reply:
Fixed in latest commit